config/locales/controllers/admin/fr.yml

```
@@ -1,18 +1,153 @@
+      wrong: Something went wrong with deleting the broadcast.
+    consumer_apps_controller:
+      created: "%{app} has been created!"
+      deleted: "%{app} has been deleted!"
+      updated: "%{app} has been updated!"
+      wrong: Something went wrong with deleting %{app}.
+    display_ads_controller:
+      created: Display Ad has been created!
+      deleted: Display Ad has been deleted!
+      updated: Display Ad has been updated!
+      wrong: Something went wrong with deleting the Display Ad.
     extensions_controller:
       update_success: Les extensions ont été mises à jour.
+    html_variants_controller:
+      fork: "%{name} FORK-%{rand}"
+      created: HTML Variant has been created!
+      deleted: HTML Variant has been deleted!
+      updated: HTML Variant has been updated!
+      wrong: Something went wrong with deleting the HTML Variant.
     invitations_controller:
+      duplicate: 'Invitation was not sent. There is already a registered user with the email: %{email}'
       create_success: The invite has been sent to the user's email.
       destroy_success: Invite cancelled for %{email}.
       resend_success: Invite resent to %{email}.
+    listing_categories_controller:
+      created: Listing Category has been created!
+      updated: Listing Category has been updated!
+      deleted: Listing Category has been deleted!
+    listings_controller:
+      updated: Listing updated successfully
+      destroyed: "'%{title}' was destroyed successfully"
+      no_credit: Not enough available credits
+    mods_controller:
+      trusted: "%{username} now has a Trusted role!"
+    navigation_links_controller:
+      deleted: Navigation Link %{link} deleted
+      created: 'Successfully created navigation link: %{link}'
+      updated: 'Successfully updated navigation link: %{link}'
+    organization_memberships_controller:
+      not_exist: 'Organization #%{org_id} does not exist.'
+      wrong: Something went wrong with removing the user from %{org}
+      added: User was successfully added to %{org}
+      removed: User was successfully removed from %{org}
+      updated: User was successfully updated to %{type}
+    organizations_controller:
+      credit_updated: Successfully updated credits
+    pages_controller:
+      updated: Page has been successfully updated.
+      created: Page has been successfully created.
+      deleted: Page has been successfully deleted.
+      code_of_conduct:
+        title: Code of Conduct
+        description: A page that describes how to behave on this platform
+      privacy_policy:
+        title: Privacy Policy
+        description: A page that describes the privacy policy
+      terms_of_use:
+        title: Terms of Use
+        description: A page that describes the terms of use for the application
+    podcasts_controller:
+      new_owner_added: New owner added!
+      no_such_user: No such user
+      scheduled: 'Podcast''s episodes fetching was scheduled (%{title}, #%{id})'
```

One, final suggestion (same as above) that was missed!:
```suggestion
      scheduled: "Podcast's episodes fetching was scheduled (%{title}, #%{id})"
```